CONTRIBUTING.md

```diff
@@ -15,24 +14,24 @@ inclusion.
 ## Pull Request Process
 
 1. Features should generally be pulled against the `master` branch.
-   Bug-fixes, depending on the urgency and risk, goes either against an
-   active release-branch, or `master`.
+   Bug-fixes, depending on the urgency and risk, go either against an active
+   release-branch, or `master`.
 2. Make sure the code follows our [coding-style](Documentation/CodingStyle.md).
 3. New features should have tests added, ensuring the feature does not
-   silently break. Bug-fixes generally should have a regression-test added
-   to ensure it isn't reintroduced.
+   silently break. Bug fixes should generally have regression tests added
+   to ensure they're not reintroduced later.
 4. New features must be documented. Please read [this](Documentation/WritingDocumentation.md)
    for details on how to write documentation.
-5. If there's any functional changes, update CHANGELOG.md with a brief
+5. If there are any functional changes, update CHANGELOG.md with a brief
    explanation.
-6. Ensure that tests pass locally before sending a pull-request.
+6. Ensure that tests passes locally before sending a pull-request.
```

Comment:
"pass" is correct in this case. My comment in the other PR was only meant to cover the next one.